portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,14 +19,45 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
-
+import java.util.ArrayList;
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+    
+    
+
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("Hello Juan!");
+    ArrayList<String> jsonarray = new ArrayList<String>();
+    jsonarray.add("I love this site.");
+    jsonarray.add("This is so cool.");
+    jsonarray.add("All the cool kids are coding nowadays.");
+    // response.setContentType("text/html;");
+    // response.getWriter().println("Hello Juan!");
+
+    String json = convertToJson(jsonarray);
```

Comment:
Very admirable you wrote a whole function to do this yourself, but you can just reuse Gson to make things easier :) Check out this example: https://github.com/John256145/software-product-sprint/blob/master/walkthroughs/week-2-server/examples/todo-list/src/main/java/com/google/sps/servlets/ListTasksServlet.java (starting at line 54)

Reply:
My portfolio does not use json anymore so I deleted the function instead.